docs-ref-conceptual/manage-cloud-sets-azure-cli.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Managing Cloud Sets with Azure CLI 2.0
+description: Create, log in, and manage different cloud sets with the Azure CLI 2.0.
+keywords: Azure CLI 2.0, Azure, cloud sets, datacenters
+cloud: ???
+author: sptramer
+manager: douge
+
+ms.author: sttramer
+ms.date: 06/27/2017
+ms.topic: article
+ms.prod: azure
+ms.technology: azure
+ms.devlang: azurecli
+ms.service: ???

```

Comment:
@rloutlaw - Unclear what service this is associated with - do we have a list available?